tvm/src/codegen/codegen_source_base.cc

```diff
@@ -116,5 +116,13 @@ void CodeGenSourceBase::EndScope(int scope_id) {
   indent_ -= 2;
 }
 
+void CodeGenSourceBase::SetIndent(int indent) {
+  indent_ = indent;
+}
+
+int CodeGenSourceBase::GetIndent() {
+  return indent_;
+}
+
```

Comment:
Why do we need this get/set method? We have already had `indent_` plus or minus 2.

Reply:
We need this get/set method to restore and set indent to zero. SystemC backend needs to generate multiple files in one IR traversal